wdio.conf.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const defaultWdioConfig = require('terra-toolkit/config/wdio/wdio.conf');
+
+const wdioConfig = defaultWdioConfig.config;
+
+const travis = process.env.TRAVIS;
+
+if (travis) {
+  wdioConfig.host = 'localhost';
```

Comment:
totally curios, if you set this value to be localhost but enable the selenium-docker-service, shoudln't this work correctly? I'm assuming you did try that..I'm just a little confused why this didn't work.

Reply:
that would be equivalent to disabling the CI env and running locally. It seemed like the selenium docker service had trouble starting up and would time out every time.